sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2_and_later/async_samples/sample_send_request_async.py

```diff
@@ -0,0 +1,83 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_send_request_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to make custom HTTP requests through a client pipeline.
+
+USAGE:
+    python sample_send_request_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Form Recognizer resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer API key
+"""
+
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.core.rest import HttpRequest
+from azure.ai.formrecognizer.aio import DocumentModelAdministrationClient
+
+
+async def sample_send_request():
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+
+    client = DocumentModelAdministrationClient(
+        endpoint=endpoint, credential=AzureKeyCredential(key)
+    )
+    
+    # The `send_request` method can send custom HTTP requests that share the client's existing pipeline,
+    # while adding convenience for endpoint construction and service API versioning.
+    # Now let's use the `send_request` method to make a resource details fetching request.
+    # The URL of the request can be relative (your endpoint is the default base URL),
+    # and the API version of your client will automatically be used for the request.
+    request = HttpRequest(method="GET", url="info")
+    async with client:
+        response = await client.send_request(request)
+    
+    response_body = response.json()
+    print(
+        f"Our resource has {response_body['customDocumentModels']['count']} custom models, "
+        f"and we can have at most {response_body['customDocumentModels']['limit']} custom models."
+        f"The quota limit for custom neural document models is {response_body['customNeuralDocumentModelBuilds']['quota']} and the resource has"
+        f"used {response_body['customNeuralDocumentModelBuilds']['used']}. The resource quota will reset on {response_body['customNeuralDocumentModelBuilds']['quotaResetDateTime']}"
+    )
+
+
+async def main():
+    await sample_send_request()
+
+
+if __name__ == "__main__":
+    import sys
+    from azure.core.exceptions import HttpResponseError
+
+    try:
+        asyncio.run(main())
+    except HttpResponseError as error:
```

Comment:
An `HttpResponseError` won't be raised by `send_request` unless you call `raise_for_status()` on the returned response, so this block is currently unnecessary. We can probably keep it though and add a `raise_for_status()` check to the sample code